packages/@dcl/build-ecs/index.ts

```diff
@@ -824,14 +828,16 @@ function validatePackageJsonForLibrary(
     }
   }
 
-  if (!packageJson.typings) {
-    throw new Error(`field "typings" in package.json is missing.`)
+  if (!packageJson.typings && !packageJson.types) {
+    throw new Error(`field "typings" or "types" in package.json is missing.`)
   } else {
-    const typingsFile = ts.sys.resolvePath(packageJson.typings)
+    const typingsFile = ts.sys.resolvePath(
+      (packageJson.typings || packageJson.types) as string
```

Comment:
what if u have both keys ?

Reply:
If you have both, it prefers `typings` to `types`, this is only a problem if they're different, and if they are, then I would say that's a problem with the published package!